### PR TITLE
added two buttons on the homepage and a pop up message on the map page

### LINKED ISSUE #67: Add two buttons: one for in-person events and another for both in-person and online events
The 'In-person events' button directs to the Map page, and the 'both in-person and online events' button directs to the List View page.

Example below

Image

### PR DESCRIPTION
**Pull request by SachiGoto**
### Related Issue
Added two buttons: one directs you to the map page, and the other one directs you to the list view page.

Screenshot 2023-10-13 at 5 58 55 PM

Added a popup message on the map that displays the number of events on the selected day.
giphy

Issue Number
close #67 #74 

### Something you want to mention
It doesn't have a smooth transition from 'no events' to 'there is 1 event on that day' popup message. I need to figure out how to render pop up in a proper order.... 

### Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/app/map/components/GoogleMaps/index.tsx [hunk 1]
```diff
@@ -148,9 +154,22 @@ export const GoogleMaps = ({ setMapLoaded }: GoogleMapsProps) => {
...
+    }else {
+        setEvents(false);
+        setNoEvents(true);
+      }
     }
   }, [loading, error, result, infoWindowID]);
 
+    const [noEvents, setNoEvents] = useState(false);
+    const [events, setEvents] = useState(false);
+
```
Comment (keento0809) [inline src/app/map/components/GoogleMaps/index.tsx:163]: @SachiGoto I feel like we can check if there are events or not with only one boolean state value.

**Review by keento0809 — COMMENTED** [review]
@SachiGoto Thank you for your contribution! Codes look good to me, and I just left a comment for enhancement of codes!

File: src/app/map/components/GoogleMaps/index.tsx (hunk 1 above)
Comment (SachiGoto) [reply src/app/map/components/GoogleMaps/index.tsx:163]: ohhh wait a minute, that's true!! I will refactor the code!

Comment (SachiGoto) [reply src/app/map/components/GoogleMaps/index.tsx:163]: Just refactored the code. It has smooth transition of popup messages too!

Comment (keento0809) [reply src/app/map/components/GoogleMaps/index.tsx:163]: Nicely done!! 🙌

**Review by keento0809 — APPROVED** [review]

**Review by devjoylee — APPROVED** [review]
LGTM

**Pull request merged**